### PR TITLE
Fix/15681618 team reputation

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15681618

Changes in this PR:
- rounded percentage of teams reputation
- added tooltip on hover to show full percentage value

image
image

### CODE REVIEW

**Review by melyndav — APPROVED** [review]
Great work on this @CzarekDryl. Approved

**Review by arrenv — APPROVED** [review]
Looks good @CzarekDryl. thank you for update.

**Review by rdig — CHANGES_REQUESTED** [review]
All good, however the "All Other" entries shows the tooltip rendered to the left, ranther than on top

Screenshot from 2023-11-27 16-47-56
Screenshot from 2023-11-27 16-48-00

**Comment by CzarekDryl** [comment]
@rdig Thanks for checking that, I've fixed this issue.

File: src/components/v5/common/TeamReputationSummary/partials/TeamReputationSummaryRow.tsx [hunk 1]
```diff
@@ -1,15 +1,14 @@
 import React, { FC } from 'react';
 
-import { setTeamColor } from '../utils';
+import { formatPercentage, setTeamColor } from '../utils';
 import { TeamPointsRowProps } from '../types';
-import Numeral from '~shared/Numeral';
+import Tooltip from '~shared/Extensions/Tooltip';
```
Comment (ArmandoGraterol) [inline src/components/v5/common/TeamReputationSummary/partials/TeamReputationSummaryRow.tsx:5]: Same here

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me 👍 Thank you for your work

**Review by arrenv — DISMISSED** [review]
Looks consistent now, thanks @CzarekDryl

**Review by arrenv — APPROVED** [review]

**Review by melyndav — DISMISSED** [review]
Nice work @CzarekDryl! Approved :)

**Review by arrenv — APPROVED** [review]
Thanks @CzarekDryl, looks good to me!

**Review by jakubcolony — DISMISSED** [review]
Nice work 👍

**Review by arrenv — APPROVED** [review]

**Review by TomaszChmielPagepro — APPROVED** [review]

**Review by jakubcolony — APPROVED** [review]

**Pull request merged**